web/test/karma.conf.js

```diff
@@ -40,6 +40,8 @@ module.exports = function(config) {
       'bower_components/angular-bootstrap/ui-bootstrap-tpls.js',
       'bower_components/lodash/lodash.js',
       'bower_components/angular-uuid/uuid.min.js',
+      'bower_components/leaflet/dist/leaflet.js',
```

Comment:
Haha, I just had the same thing happen to me. The exclude was also needed on the 'test' section. I've added that and the line has stopped resurrecting itself.